.golangci.yml

```diff
@@ -0,0 +1,3 @@
+linters:
```

Comment:
Temporarily added because go vet points out an issue with `ServiceDocumentSource`.
Right now it looks like:
```go
# Embedded, both `IntegrationId` and `ServiceRepository` have a clashing `json:"id"` tag
type ServiceDocumentSource struct {
	IntegrationId     `graphql:"... on ApiDocIntegration"`
	ServiceRepository `graphql:"... on ServiceRepository"`
}
```
but it should look like
```go
type ServiceDocumentSource struct {
	IntegrationId     IntegrationId     `graphql:"... on ApiDocIntegration"`
	ServiceRepository ServiceRepository `graphql:"... on ServiceRepository"`
}
```